### DISCUSSION TITLE
Relative file paths from markdown in generated html

### DISCUSSION THREAD

**Discussion by BerendPijnenburg** [doxygen/doxygen#11080] [Q&A] [answered]
I have a README with a relative link to a file using [ myfile ] (doc/myfile)
When i generate html with doxygen the path link points to pathToRootDir/doc/html/doc/myfile instead of the desired path of pathToRootDir/doc/myfile. Is there a way to let all relative paths start from the root directory of a project or let doxygen generate the relative paths from the source file location and how do I go about doing this?

**Comment by albert-github** [comment]
- Which version of doxygen did you use?

When I understand it correctly the `pathToRootDir/doc/myfile` is the original file and the file `pathToRootDir/doc/html/doc/myfile` is the place where a copy of the file is located after running doxygen. 
In my opinion this is correct so the result (`pathToRootDir/doc/html/doc/myfile`) is internally consistent, especially important when copying later on the resulting "html" tree to a server that cannot access the original  "pathToRootDir".

**Reply by BerendPijnenburg** [reply]
I'm using doxygen 1.11.0, linking to generated copies in the html works perfectly fine.
My situation is that I have some links to configuration files in my markdown documents to which I want to link instead of a copy in our internal documentation. I understand it is not the intended use but was wondering if it is possible to let doxygen generate paths to the source file using the paths specified in the markdown.

**Reply by albert-github** [reply, marked as answer]
> I want to link instead of a copy in our internal documentation.

Did you have a look at `GENERATE_TAGFILE` and `TAGFILES`?
To give an idea of how a tag file looks like (for your internal documentation):
```
<?xml version='1.0' encoding='UTF-8' standalone='yes' ?>
<tagfile doxygen_version="1.13.0" doxygen_gitid="c6f687c0a8092fbe24a153e64b754f971b2f4cb6*">
  <compound kind="page">
    <name>md_aa</name>
    <title>aa</title>
    <filename>md_aa.html</filename>
    <docanchor file="md_aa.html">the_anchor</docanchor>
  </compound>
</tagfile>
```
which has been generated by means of:
**aa.md**
```
Test on anchor in tagfile

\anchor the_anchor
```

**Doxyfile**
```
QUIET = YES
GENERATE_TAGFILE = mytag.tag
```

and now you can use in your own documentation:
```
\ref the_anchor
```
of course when you set `TAGFILES=... ` properly.

The used  example: [example.tar.gz](https://github.com/user-attachments/files/16663091/example.tar.gz)

**Reply by BerendPijnenburg** [reply]
Thanks this was the way to go.
